logic/atrium-logic/src/commonMain/kotlin/ch/tutteli/atrium/logic/impl/DefaultAnyAssertions.kt

```diff
@@ -18,8 +17,17 @@ class DefaultAnyAssertions : AnyAssertions {
     override fun <T> toBe(container: AssertionContainer<T>, expected: T): Assertion =
         container.createDescriptiveAssertion(TO_EQUAL, expected) { it == expected }
 
-    override fun <T> notToBe(container: AssertionContainer<T>, expected: T): Assertion =
-        container.createDescriptiveAssertion(NOT_TO_EQUAL, expected) { it != expected }
+    override fun <T> notToBe(container: AssertionContainer<T?>, expected: T?): Assertion {
```

Comment:
There isn't a need to change `expected: T`  to `expected: T?` because `T` has not `Any`  as upper bound constraint, i.e. it is already possible to use e.g. `String?` as substitute for `T, thus one can already pass `null`  as expected since `null`  is a valid value of ` String?` 
```suggestion
    override fun <T> notToBe(container: AssertionContainer<T>, expected: T): Assertion {
```